airbyte-webapp/src/config/types.ts

```diff
@@ -13,20 +13,36 @@ declare global {
     REACT_APP_WEBAPP_TAG?: string;
     REACT_APP_INTERCOM_APP_ID?: string;
     REACT_APP_INTEGRATION_DOCS_URLS?: string;
-    SEGMENT_TOKEN?: string;
+    SEGMENT_TOKEN: string;
```

Comment:
As far as I can tell, we have segment enabled in every environment. For example, in OSS we set `REACT_APP_SEGMENT_TOKEN` in `airbyte-webapp/.env`

Reply:
While the token should always be set (and I would prefer we even don't rely on this), this here is the typing for the `window` object. We're not having the SEGMENT_TOKEN always set on the `window` object, since e.g. in OSS it's coming in via the `.env` file and thus `process.env`.

Reply:
Right, then my question is irrelevant, whoops. I'll make it optional on `window` again.